### ISSUE TITLE
Simple GUI

### ISSUE THREAD

**Issue by Majekdor** [MajekDev/HexNicks#50]
Add a simple command that opens a GUI that allows players to basically execute /nickcolor. Use blocks of wool to show the default MC color codes and when you click it'll set your nick color to that. 

Potentially add something for hex too.

**Comment by funnyboy-roks** [comment]
I got bored and decided to play around with a GUI a bit...

What are your thoughts on the following layout?  If you like, I can make a PR.

`/nickcolor` with no args opens this gui:

![image](https://github.com/MajekDev/HexNicks/assets/70620591/4848b1a0-dd0b-40c9-b87c-c968ad88025d)

One can click on the rainbow head to open this GUI for hexadecimal colours:

![image](https://github.com/MajekDev/HexNicks/assets/70620591/fbc1c31a-94b4-4809-a157-8cc881723e50)

The rainbow head chooses a random colour:

![image](https://github.com/MajekDev/HexNicks/assets/70620591/254a28fe-bf9e-424f-86cc-aeebc113f93e)

The nametag shows the coloured nickname and the colour that is selected.  When clicked, it updates the player's nickname.

![image](https://github.com/MajekDev/HexNicks/assets/70620591/a409cfbb-d18f-408f-8efe-a9f0aa2d65ea)

and the nether start is a "back" button to get back to the default colours

**Comment by Majekdor** [comment]
Yeah this is almost exactly what I had pictured in my head! It looks great! 
Feel free to PR :D
